packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3548,8 +3548,8 @@ class PositionedDirectional extends StatelessWidget {
 /// horizontal) or [Column] (if it's vertical) instead, because that will be less
 /// verbose.
 ///
-/// To cause a child to expand to fill the available vertical space, wrap the
-/// child in an [Expanded] widget.
+/// To cause a child to expand to fill the available space in the [direction]
```

Comment:
I don't think `direction` is a variable/in-scope identifier. Should probably just be direction or \`direction`

Reply:
https://docs.flutter.io/flutter/widgets/Flex/direction.html